backend/src/v2/compiler/argocompiler/container.go

```diff
@@ -233,6 +245,12 @@ func (c *workflowCompiler) addContainerExecutorTemplate() string {
 					MountPath: component.VolumePathKFPLauncher,
 				}},
 				Resources: launcherResources,
+				Env: []k8score.EnvVar{
+					{
+						Name:  LauncherImageEnvVar,
+						Value: GetLauncherImage(),
```

Comment:
is this value the same as `c.launcherImage` at line 241? if yes, you can directly use it. no need to call the function again, although it gets the same value.